plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveColumnHandle.java

```diff
@@ -66,7 +66,7 @@
     public static final int FILE_MODIFIED_TIME_COLUMN_INDEX = -14;
     public static final String FILE_MODIFIED_TIME_COLUMN_NAME = "$file_modified_time";
     public static final HiveType FILE_MODIFIED_TIME_TYPE = HiveType.HIVE_TIMESTAMP;
-    public static final Type FILE_MODIFIED_TIME_TYPE_SIGNATURE = TIMESTAMP_TZ_MILLIS;
+    public static final Type FILE_MODIFIED_TIME_TYPE_SIGNATURE = TIMESTAMP_TZ_MICROS;
```

Comment:
The value comes from https://github.com/trinodb/trino/blob/f6422d04663ab011d3ddb831ed16dec02659c47e/plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveUtil.java#L942 and is in millis, as filled here:

https://github.com/trinodb/trino/blob/b004f722e73ab341381316a3cb293c42260e1266/plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/InternalHiveSplitFactory.java#L126

https://github.com/trinodb/trino/blob/b004f722e73ab341381316a3cb293c42260e1266/plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/InternalHiveSplitFactory.java#L142

what if we keep TIMESTAMP_TZ_MILLIS here?